src/Select.soy

```diff
@@ -39,7 +39,7 @@
 			{param expanded: $disabled ? false : $expanded_ /}
 			{param header kind="html"}
 				<button
-					class="{$buttonClass ? $buttonClass : ''} dropdown-select"
+					class="{$buttonClass ?: 'btn btn-default'} dropdown-select"
```

Comment:
It shouldn't really re render, but what will happen is that when you do select.cssClass it will return something that is not what was used to render